core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/CommonLoadBalancer.scala

```diff
@@ -214,6 +240,18 @@ abstract class CommonLoadBalancer(config: WhiskConfig,
 
   protected def releaseInvoker(invoker: InvokerInstanceId, entry: ActivationEntry)
 
+  // Singletons for counter metrics related to completion acks
+  val LOADBALANCER_COMPLETION_ACK_REGULAR =
```

Comment:
Can be made private

Reply:
Good feedback. I made them `protected`. If a sub-type decided to override `processCompletion()`, it also had access to the `LogMarkerToken` instances.